src/components/Label/Label.md

```diff
@@ -0,0 +1,35 @@
+# Label
+Used to label form elements, such as TextField and Dropdown.
```

Comment:
Is it worthwhile to mention where the label typically goes (all of them)?

This would be:
- Choice fields (checkbox, radio button)
- Textfield
- Dropdown
- People Picker
- Toggle